templates/show_table_info.html

```
@@ -383,6 +372,24 @@ <h5><i class="fas fa-code"></i> SQL Query Editor
                                 {% endfor %}
                             </ul>
                         </div>
+                        <div class="dropdown">
+                            <button class="btn btn-outline-primary btn-sm dropdown-toggle" type="button" id="historyBtn" data-bs-toggle="dropdown" data-bs-auto-close="true" aria-expanded="false">
+                                <i class="fas fa-clock"></i> Query History
+                            </button>
+                            <ul class="dropdown-menu" aria-labelledby="historyBtn">
+                                {% if session['history'] and session['history']|length > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)